ios/RNGoogleMobileAds/RNGoogleMobileAdsAppOpenModule.m

```diff
@@ -21,6 +21,9 @@
 #import "RNGoogleMobileAdsCommon.h"
 #import "common/RNSharedUtils.h"
 
+static __strong GADAppOpenAd *appOpen;
+static __strong RNGoogleMobileAdsFullScreenContentDelegate *appOpenDelegate;
```

Comment:
Ah, but I see they moved here. I may just not have the Objective-C skills to really review this (yet) and need to research more

Reply:
Yes, I changed them to static var because other full screen ad modules declare them as static vars as well.

Reply:
Yes, I changed them to static var because other full screen ad modules declare them as static vars as well.